### DISCUSSION TITLE
unable to use query giving error

### DISCUSSION THREAD

**Discussion by loveleet-convivity** [payloadcms/payload#3368] [Q&A]
{
      type: "relationship",
      relationTo: "categoriessubitem",
      name: "child",
      admin: {
        position: "sidebar",
      },
      hasMany: false,
      filterOptions: ({ data }) => {
        console.log("hjk", data);
        return {
          'parent.id': { equals: data?.categoriesitem },
        };
      },
    },

**Comment by matthijs166** [comment]
Have you already tried something like the following? Don't think you need to reference the ID there:

```js
const field = {
  type: "relationship",
  relationTo: "categoriessubitem",
  name: "child",
  admin: {
    position: "sidebar",
  },
  hasMany: false,
  filterOptions: ({ data }) => {
    return {
      "parent": { 
        equals: data?.categoriesitem.id // or data?.categoriesitem
      },
    };
  },
};
```
